addons/snailmail/models/snailmail_letter.py

```diff
@@ -366,7 +366,9 @@ def _snailmail_print_valid_address(self):
         self.message_id._notify_message_notification_update()
 
     def snailmail_print(self):
-        self._snailmail_print()
+        self.write({'state': 'pending'})
```

Comment:
Should likely call `self.message_id._notify_message_notification_update()` after `write` to ensure the interface is properly updating

Reply:
indeed, thanks :)

Reply:
Note that I believe this would apply from 13.4.